docs/docs/babel.md

```diff
@@ -22,48 +22,26 @@ browsers.
 ## How to use a custom .babelrc file
 
 Gatsby ships with a default .babelrc setup that should work for most sites. If you'd like
```

Comment:
Is the default setup the `babel-preset-gatsby-package` thing? If so, would it make sense to link to it here and name it?

Reply:
`babel-preset-gatsby` would/will be the one used by 99% of users!

Reply:
I have yet to finish the implementation as well as the docs. Will definitely incorporate this, thanks 🙂

My plan is to have it ready for a next review cycle until the end of the week.

Reply:
@philipp-spiess whenever you can get to is great! I think this is an impactful change, particularly in re: to being easier to set up testing!